assets/js/components/ErrorNotice.js

```diff
@@ -63,9 +63,9 @@ export default function ErrorNotice( {
 		);
 	}, [ dispatch, selectorData ] );
 
-	// Do not display if there is no error, or if the error is for missing scopes.
-	// This only applies when no message prop is directly passed.
-	if ( ! message && ( ! error || isPermissionScopeError( error ) ) ) {
+	// Do not display if there is no error and no direct message text is passed as a direct prop.
+	// Also do not display if the error is for missing scopes as these are handled by a popup modal.
+	if ( ( ! error && ! message ) || isPermissionScopeError( error ) ) {
```

Comment:
Can we maybe simplify this? AFAICT, if there is no message, this would not be shown regardless if there is an error or not. `isPermissionScopeError` is also safe to use without an error.

i.e.
```suggestion
	if ( ! message || isPermissionScopeError( error ) ) {
```

Otherwise, if there is an error and no message, then this would still show (if not a permission error), which would still be incorrect I think.

Reply:
> Otherwise, if there is an error and no message, then this would still show

This makes sense. I think previously, it did exactly this - so I'm guessing there always was a message for every error. But we shouldn't rely on it.

Thanks!